src/main/java/com/aws/greengrass/componentmanager/KernelConfigResolver.java

```diff
@@ -334,12 +341,13 @@ private void updateRunWith(RunWith runWith, Map<String, Object> resolvedServiceC
      * @param configurationUpdateOperation nullable component configuration update operation.
      * @param componentRecipe              component recipe containing default configuration.
      * @param document                     deployment document
+     * @param configMergeTimestamp         timestamp to use during configuration merge
```

Comment:
nit: since you removed the document var here, remove it from the method doc as well

Reply:
Will update